CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @gsoft-inc/internal-developer-platform
+* @gsoft-inc/infrastructure
```

Comment:
is there a difference if we put it on two lines or keep both in one line? On reusable-workflows, we keep on one